[2021-08-10 22:10:29,705] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_2 2021-08-10T22:10:21.757888+00:00 [queued]>
[2021-08-10 22:10:29,795] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_2 2021-08-10T22:10:21.757888+00:00 [queued]>
[2021-08-10 22:10:29,798] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 22:10:29,799] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-10 22:10:29,799] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 22:10:29,846] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_2> on 2021-08-10T22:10:21.757888+00:00
[2021-08-10 22:10:29,856] {standard_task_runner.py:52} INFO - Started process 333880 to run task
[2021-08-10 22:10:29,885] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_2', '2021-08-10T22:10:21.757888+00:00', '--job-id', '79', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpwbgjb6tg', '--error-file', '/tmp/tmpil4hjt27']
[2021-08-10 22:10:29,920] {standard_task_runner.py:77} INFO - Job 79: Subtask task_2
[2021-08-10 22:10:30,313] {logging_mixin.py:104} INFO - Running <TaskInstance: trigger_rule.task_2 2021-08-10T22:10:21.757888+00:00 [running]> on host airflowvm
[2021-08-10 22:10:30,685] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T22:10:21.757888+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T22:10:21.757888+00:00
[2021-08-10 22:10:30,694] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-10 22:10:30,695] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'exit 0']
[2021-08-10 22:10:30,725] {subprocess.py:75} INFO - Output:
[2021-08-10 22:10:30,726] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-10 22:10:30,808] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_2, execution_date=20210810T221021, start_date=20210810T221029, end_date=20210810T221030
[2021-08-10 22:10:31,011] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-10 22:10:31,034] {local_task_job.py:151} INFO - Task exited with return code 0
